httpserver-rs/src/lib.rs

```diff
@@ -191,7 +191,16 @@ impl HttpServerCore {
                     let span = tracing::debug_span!("http request", %method, path = %path.as_str(), %query);
                     let ld = linkdefs.clone();
                     let arc_inner = arc_inner.clone();
-                    async move {
+                    async move{
+                        if let Some(mode) = ld.values.get("READONLY_MODE"){
```

Comment:
Should this be part of `ServiceSettings`?

Reply:
If possible, it would be nice to retrieve both the upper and lowercase value here. If that's too verbose, then I would recommend the lowercase since the other settings use that as well.

Can we also add this to settings.md?